.github/workflows/node.js.yml

```diff
@@ -23,6 +23,6 @@ jobs:
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@releases/v3
         with:
-          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This is provided by GitHub.
```

Comment:
This thing is not provided by Github we need to add a secret for that with all the check for deployment

Reply:
No, it is provided I have checked it in many repos.

Reply:
You can check our latest deployment it's failing because of that

Reply:
It should like this we forgot to change the key 
ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This is provided by GitHub.

Reply:
Ref : https://github.com/JamesIves/github-pages-deploy-action/issues/231#issuecomment-628599252